sentry-android-core/src/main/java/io/sentry/android/core/ContextUtils.java

```diff
@@ -202,20 +233,10 @@ static boolean isForegroundImportance(final @NotNull Context context) {
 
         // getInstallSourceInfo requires INSTALL_PACKAGES permission which is only given to system
         // apps.
+        // if it's installed via adb, system apps or untrusted sources
+        // could be amazon, google play etc - or null in case of sideload
         final String installerPackageName = packageManager.getInstallerPackageName(packageName);
-
-        final Map<String, String> sideLoadedInfo = new HashMap<>();
-
-        if (installerPackageName != null) {
-          sideLoadedInfo.put("isSideLoaded", "false");
-          // could be amazon, google play etc
-          sideLoadedInfo.put("installerStore", installerPackageName);
-        } else {
-          // if it's installed via adb, system apps or untrusted sources
-          sideLoadedInfo.put("isSideLoaded", "true");
-        }
-
-        return sideLoadedInfo;
+        return new SideLoadedInfo(installerPackageName == null, installerPackageName);
```

Comment:
You can put the check inside the constructor, so that you can pass only one parameter